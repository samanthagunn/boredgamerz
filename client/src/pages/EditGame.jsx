import { IonContent, IonHeader, IonPage } from "@ionic/react";
import React from "react";
import Footer from "../components/footer";
import Form from "../components/form";
import Header from "../components/header";

const EditGame = () => {
  let edit = true;
  return (
    <IonPage>
      <Header />
      <IonContent>
        <Form editState={edit} />
      </IonContent>
      <Footer />
    </IonPage>
  );
};

export default EditGame;
