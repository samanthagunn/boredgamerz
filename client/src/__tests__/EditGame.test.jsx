import React from "react";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import EditGame from "../pages/EditGame";
import renderer from "react-test-renderer";


jest.mock('../components/auth-header')
jest.mock('../components/form');
jest.mock('../components/mobile-fab')


test('EditGame page renders...',()=>{
    const {container} = render (<EditGame />);
    expect(container.textContent)
    .toMatch("Test component");

});

test("Matches the snapshot..", ()=>{
    const tree = renderer.create(<EditGame />).toJSON()
    expect(tree).toMatchSnapshot();
})