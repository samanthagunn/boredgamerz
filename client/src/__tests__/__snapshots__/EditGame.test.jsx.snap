// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches the snapshot.. 1`] = `
<div
  className="ion-page"
>
  <div>
    Test component
  </div>
  <ion-content>
    <div>
      Test component
    </div>
  </ion-content>
  <div>
    Test component
  </div>
  <ion-footer>
    <ion-toolbar
      color="tertiary"
    >
      <div>
        <p>
          Â© 2020 Bored Gamerz, BravoBunch
        </p>
      </div>
    </ion-toolbar>
  </ion-footer>
</div>
`;
